packages/eligibility-service/src/blockchain/sendActivationMessage.js

```diff
@@ -1,19 +1,22 @@
 const {BurstValue} = require("@burstjs/util")
-const {getAccountIdFromPublicKey, generateMasterKeys} = require("@burstjs/crypto")
+const {getAccountIdFromPublicKey, generateMasterKeys, convertNumericIdToAddress} = require("@burstjs/crypto")
 const {AttachmentMessage} = require('@burstjs/core')
 const Config = require('../config')
 const {logger} = require("../logger");
 const {BurstApi} = require("./burstApi")
 
-async function sendActivationMessage({recipientPublicKey, activationPassphrase, votingOptions}) {
+async function sendActivationMessage({recipientPublicKey, activationPassphrase, votingOptions, votingPassphrase}) {
     logger.info(`Sending Activation Message to ${recipientPublicKey}`)
 
     const recipientId = getAccountIdFromPublicKey(recipientPublicKey);
     const senderKeys = generateMasterKeys(activationPassphrase);
 
+    const votingAccKeys = generateMasterKeys(votingPassphrase);
+    const votingAddress = convertNumericIdToAddress(getAccountIdFromPublicKey(votingAccKeys.publicKey));
+
     const attachment = new AttachmentMessage({
         messageIsText: true,
-        message: JSON.stringify(votingOptions)
+        message: JSON.stringify(votingOptions.push({ votingAddress: votingAddress })) //I am not sure about this
```

Comment:
lets create a json like this:
```js
{
 voops: [... voting options]
vaddr: "2368724999034834"
}
```

Then stringify the JSON and attach to message